piksi_tools/serial_link.py

```diff
@@ -31,7 +31,7 @@
 from sbp.logging import SBP_MSG_LOG, SBP_MSG_PRINT_DEP, MsgLog
 from sbp.piksi import MsgReset
 
-from piksi_tools.utils import mkdir_p
+from piksi_tools.utils import (mkdir_p, get_tcp_driver)
```

Comment:
is there a reason for the parentheses? Only asking because inconsistent style with above